### PR TITLE
Transfer Terms to Angular

### PR DESCRIPTION
**Pull request by wpreimes**

### CODE REVIEW

**Comment by wpreimes** [comment]
@sheenaze not sure why the  UI/package-lock.json  file has changed? Otherwise this should work.

**Comment by wpreimes** [comment]
ok, there are a few more issues with css still ;)

File: UI/src/app/pages/terms/terms.component.html [hunk 1]
```diff
@@ -0,0 +1,109 @@
+<link rel="stylesheet" href="../about/about.component.scss">
```
Comment (sheenaze) [inline UI/src/app/pages/terms/terms.component.html:1]: if you want to load styling you should do that in scss file, not in html one

File: UI/src/app/pages/terms/terms.component.html [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+<script src="terms.component.ts"></script>
```
Comment (sheenaze) [inline UI/src/app/pages/terms/terms.component.html:2]: the same here, you don't need to load it, angular knows which .ts file is combined to this html.

File: UI/src/app/pages/terms/terms.component.html [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+<link rel="stylesheet" href="terms.component.scss">
```
Comment (sheenaze) [inline UI/src/app/pages/terms/terms.component.html:3]: same here

**Review by sheenaze — APPROVED** [review]

**Pull request merged**